packages/admin/admin/src/dataGrid/CrudContextMenu.tsx

```diff
@@ -0,0 +1,193 @@
+import { ApolloClient, RefetchQueriesOptions, useApolloClient } from "@apollo/client";
+import { Copy, Delete as DeleteIcon, Domain, MoreVertical, Paste, ThreeDotSaving } from "@comet/admin-icons";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, IconButton, ListItemIcon, ListItemText, Menu, MenuItem } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { readClipboardText } from "../clipboard/readClipboardText";
+import { writeClipboardText } from "../clipboard/writeClipboardText";
+import { useErrorDialog } from "../error/errordialog/useErrorDialog";
+
+interface DeleteDialogProps {
+    dialogOpen: boolean;
+    handleNoClick: () => void;
+    handleYesClick: () => void;
```

Comment:
I'd call those two callbacks `onDelete` and `onCancel`. If the text of the buttons change, we would need to update the callbacks too.